packages/airnode-node/src/api/index.ts

```diff
@@ -76,14 +86,36 @@ async function signResponseMessage(requestId: string, responseValue: string, con
   );
 }
 
+async function signRelayedResponseMessage(
+  requestId: string,
+  timestamp: string,
+  relayerAddress: string,
+  responseValue: string,
+  config: Config
+) {
+  const masterHDNode = getMasterHDNode(config);
+  const airnodeWallet = ethers.Wallet.fromMnemonic(masterHDNode.mnemonic!.phrase);
+
+  return await airnodeWallet.signMessage(
```

Comment:
Based on https://github.com/api3dao/airnode/blob/v1-protocol/packages/airnode-protocol-v1/contracts/dapis/DapiServer.sol#L369

Reply:
I think you mean https://github.com/api3dao/airnode/blob/v1-protocol/packages/airnode-protocol-v1/contracts/protocol/AirnodeProtocol.sol#L300 and the order is wrong

Reply:
🤦 sorry for the order mistake - I was careless.